redisenterprise/README.md

```diff
@@ -37,7 +37,23 @@ This integration is provided by Redis Labs.
 
 ## Setup
 
-Copy the [sample configuration][2] and update the required sections to collect data from your Redis Enterprise cluster
+### Installation
+
+If you are using Agent v6.8+ follow the instructions below to install the RedisEnterprise check on your host. See the dedicated Agent guide for [installing community integrations][3] to install checks with the [Agent prior v6.8][4] or the [Docker Agent][5]:
```

Comment:
The target page for these links has changed quite a bit. See https://docs.datadoghq.com/agent/guide/use-community-integrations Not sure if "v6.8" should be changed to "v7.21/v6.21" in this sentence. but the link targets for links 3, 4, and 5 are: 
- https://docs.datadoghq.com/agent/guide/use-community-integrations/
- https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=agentearlierversions
- https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=docker